high_availability_guide/_topics/installation.md

```diff
@@ -120,11 +106,9 @@ and secondary database-only appliances can be configured.
 
 5.  In the appliance console, configure the following:
 
-    1.  Configure the hostname by selecting **Set Hostname**.
-
-    2.  Select **Configure Application**.
+    1.  Select **Configure Application**.
 
-    3.  Configure this appliance to use the encryption key from the
+    2.  Configure this appliance to use the encryption key from the
         primary database-only appliance:
 
         1.  For **Encryption Key**, select **Fetch key from remote
```

Comment:
Fetch key from remote should be in **.

This should be select **Fetch key from remote**.

Reply:
@qahmed1998 I think all of these changes are "wrong" to start with.  I'm wondering if we should have these changes done afterwards.

Reply:
Ignore my comment for fetch key from remote. This is fine.